src/query/server/cost_reporters.go

```diff
@@ -79,8 +79,10 @@ func newConfiguredChainedEnforcer(
 	globalReporterScope := globalScope.SubScope(reporterScopeName)
 
 	globalLimitMgr := cost.NewStaticLimitManager(
-		cfg.Limits.Global.AsLimitManagerOptions().
-			SetInstrumentOptions(instrumentOptions.SetMetricsScope(globalLimitManagerScope)))
+		cost.NewLimitManagerOptions().SetDefaultLimit(cost.Limit{
```

Comment:
I left these since they report metrics as well as optionally enforce limits. We can remove this logic altogether though if the metrics aren't useful now w/ no limits present.

Reply:
Can we cut out everything here that's not just adding metrics then? All the `queriesOverLimitEnabled` etc stuff can go and we can just keep the base ones? Otherwise this thing is still quite large and intricate

Reply:
Deprecated all cost reporting code 👍